package.json

```diff
@@ -16,36 +19,34 @@
   "dependencies": {
     "@types/nodemailer": "^4.6.0",
     "babel-polyfill": "^6.26.0",
-    "body-parser": "^1.18.2",
+    "body-parser": "^1.18.3",
     "commander": "^2.15.1",
-    "cosmiconfig": "^5.0.3",
-    "express": "^4.16.2",
-    "express-validator": "^5.0.2",
-    "http-status-codes": "1.3.0",
+    "cosmiconfig": "^5.0.4",
+    "express": "^4.16.3",
+    "express-validator": "^5.2.0",
+    "http-status-codes": "^1.3.0",
     "moment": "^2.22.1",
-    "mongodb": "^3.0.4",
-    "mongoose": "^5.0.11",
+    "mongoose": "^5.1.1",
     "morgan": "^1.9.0",
-    "nodemailer": "^4.6.0",
-    "pug": "^2.0.3",
-    "pug-cli": "^1.0.0-alpha6"
```

Comment:
I removed this, so we only have stable versions of npm packages running on our server...

Reply:
The development of the cli also seems a bit abandoned